docs/doc/11-integrations/30-access-tool/01-bendsql.md

```diff
@@ -55,8 +55,21 @@ Connected to DatabendQuery v1.1.2-nightly-8ade21e4669e0a2cc100615247705feacdf76c
 bendsql>
 ```
 
+To connect to Databend Cloud, it is recommended to use the `--dsn` option or the `BENDSQL_DSN` environment variable:
```

Comment:
Can we provide the full script "bendsql --dsn xxxxxxxxxxxx" here so that users can copy and paste to their terminal?

![RaMjXMNXiM](https://user-images.githubusercontent.com/106025534/234440082-fea65e90-62e3-45e1-a90e-b2f782b31e15.jpg)